invenio_records_resources/factories/factory.py

```diff
@@ -78,6 +81,8 @@ def __init__(
         self.record_relations = record_relations
         self.schema_path = self._build_schema_path(schema_path)
         self.index_name = self._build_index_name(index_name)
+        self.model_cls_attrs = model_cls_attrs or {}
+        self.extra_record_cls_attributes = record_cls_attrs or {}
```

Comment:
```suggestion
        self.record_cls_attrs = record_cls_attrs or {}
```

I thought the different naming was because of a name-clash, but it seems that there's `self.record_cls_attrs`

Reply:
Fixed, I just wanted to avoid having `self.record_cls_attrs` and `record_class_attributes` (which exists below). And named it extra because they are actually added (not overwritting). But name changed, is all almost the same in the end, and your proposal is consistent with the other names.